apps/web/src/components/Calendar/header/date/calendar-header-date-badge.tsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react'
 import { useCalendarContext } from '../../calendar-context'
 import { sameMonth } from '../../calendar-util'
+import { useTranslation } from 'react-i18next'
 
 export default function CalendarHeaderDateBadge () {
   const { events, date } = useCalendarContext()
   const monthEvents = events.filter((event) => sameMonth(event.start, date))
+  const { t } = useTranslation()
 
   return (
     <div className='whitespace-nowrap rounded-sm border px-1.5 py-0.5 text-xs'>
-      {monthEvents.length} events
+      {`${monthEvents.length === 0 ? t('No') : monthEvents.length} ${t('Events')}`}
```

Comment:
there is a better way to have the translation handle pluralization, but i wont make you do it right now :) 

e.g.
"voterCount_one": "with {{count}} vote",
  "voterCount_other": "with {{count}} votes",